src/components/SQForm/SQFormCheckboxGroup.tsx

```diff
@@ -1,15 +1,41 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import {useFormikContext} from 'formik';
 import FormControl from '@material-ui/core/FormControl';
 import FormLabel from '@material-ui/core/FormLabel';
 import Grid from '@material-ui/core/Grid';
 import FormHelperText from '@material-ui/core/FormHelperText';
 import FormGroup from '@material-ui/core/FormGroup';
+import {CheckboxProps} from '@material-ui/core';
 import SQFormCheckboxGroupItem from './SQFormCheckboxGroupItem';
 import SQFormCheckbox from './SQFormCheckbox';
 import {useForm} from './useForm';
 
+interface CheckboxOption {
+  label: string;
+  value: unknown;
```

Comment:
I'm pretty sure, like our dropdown options, we only allow number, string, or boolean for value. maybe not though 🤔

Reply:
Ok, I'll try to narrow it and see if it breaks anything